org_chart.js

```diff
@@ -1,60 +1,33 @@
-let elon = {
-  name: "Elon Musk",
-  email: "elon.musk@musky.com",
-  position: "Boss",
-}
-
-let new_hires = [
-  {
-    name: "Alan Turing",
-    email: "alan.turing@musky.com",
-    position: "Computer Scientist",
-    project: "Cryptography"
-  },
-  {
-    name: "Grace Hopper",
-    email: "grace.hopper@musky.com",
-    position: "Computer Scientist",
-    project: "Compilers"
-  },
-  {
-    name: "Donald Knuth",
-    email: "donald.knuth@musky.com",
-    position: "Mathematician",
-    project: "Algorithm Analysis"
-  }
-]
-
-// assign each a manager
-let [alan, grace, donald] = new_hires
-
-alan.manager = elon
-grace.manager = elon
-donald.manager = elon
 
-// send to training
-if (!alan.skills) {
-  alan.skills = []
+//Person constructor
+function Person(name, email, position, project, pronoun) {
+  this.name = name;
+  this.email = email;
+  this.position = position;
+  this.project = project;
+  this.skills = [];
+  this.pronoun = pronoun;
 }
-alan.skills.push("Mathematics")
 
-if (!grace.skills) {
-  grace.skills = []
+//Set skills
+Person.prototype.addSkills = function(skill) {
```

Comment:
Since we're just adding one skill, I would suggest calling it `addSkill` instead